### PR TITLE
hello rails backend

### PR DESCRIPTION
**Pull request by Armynerh**
Created a new Rails API app called 'hello-rails-back-end'.
Initialise project with Git.
Made sure that project has a Postgres database set up.
Created a table for storing messages and created 5 different greetings.
Created an API endpoint that selects a random greeting from the table

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Kenny, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure to add a professional ReadMe to this project via [this link](https://github.com/microverseinc/readme-template), which should comply with the following:

- ReadMe should include the GitHub repo link of the _**React frontend**_ in the ReadMe of this API backend, and a GitHub repo link of this _**API backend**_ in the ReadMe of the React frontend repo.

- Kindly ensure that other aspects of your ReadMe are professionally documented:
  * Ensure that provided placeholder texts/ guide instructions are removed and/or replaced with appropriate contents applicable under each section of your ReadMe. Similarly, kindly also remove guide notes for _license_ creation under the _license_ section.
  * All main headers from the [ReadMe template](https://github.com/microverseinc/readme-template) should be present in your ReadMe, only the _FAQ_ section is optional.
  * Ensure all headers have appropriate content below describing them.
  * The _Future Features_ section of your ReadMe includes 1-3 potential features to be added later.
  * Remove the Live demo link from the ReadMe if the app is not yet deployed.
  *  Ensure to remove the Microverse template logo. Optionally, you can replace it with your own logo if you have one.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Armynerh** [comment]
Screenshot 2024-01-09 at 23 18 53
@rloterh Thank you for you feedback. However, I did include the links as required

File: README.md [hunk 1]
```diff
@@ -1,24 +1,181 @@
...
+  </ul>
+</details>
 
-* ...
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Hello World]**
+- [Frontend](https://github.com/Armynerh/hello-react-front-end)
```
Comment (Armynerh) [inline README.md:49]: @rloterh  Here is the link to the frontend

Comment (rloterh) [reply README.md:49]: Hi @Armynerh 👋 

Apparently, the branch with which this Pull Request was created does not have updated ReadMe: https://github.com/Armynerh/hello-rails-back-end/tree/feature/greeting-api

Can you please check it once again and update the ReadMe of the _feature/greeting-api_ branch?

Cheers! 😃

**Review by LionRouge1 — APPROVED** [review]
Hi @Armynerh, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

## Highlights
- Great job, your project is error free :+1: 
- Nice work you meet all the requirements :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (rloterh) [reply README.md:49]: Hi @Armynerh 

I rechecked the above link in another browser and ReadMe was displaying correctly. The was with Edge browser, which is still showing unupdated link, even after refreshing several times.

**Page displaying incorrectly in Edge browser:**
image

Apologies for any inconvenience. 🙏 

Cheers!

**Pull request merged**